authserver/api/static/css/style.css

```diff
@@ -1,51 +1,80 @@
-html,
-body {
-  height: 100%;
+:root {
+  --blue: #0fbaa8;
+  --light-blue: #0fbaa8b3;
+  --gray20: #333333;
 }
 
 body {
-  display: -ms-flexbox;
-  display: -webkit-box;
+  font-family: "Montserrat", sans-serif;
+  font-size: 18px;
```

Comment:
should remove this font-size so that it's not setting all fonts as default 18px. the default font-size is usually 16px which will be handled by setting html font-size 100%

Reply:
Ah...good call. I think the mockups ask for 18px, but using the default (html with font-size 100%) looks better and makes more sense.

Reply:
done:
https://github.com/brighthive/authserver/blob/e0dd4b2c3ca15c36f48e1af8562fdf0b772e14b3/authserver/api/static/css/style.css#L9